### DISCUSSION TITLE
Add an option to see controls in editor exactly like in the game.

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4361] [Editor]
When I create UI, like game menu etc, the theme of controls in editor is different than in game, for example some icons are different.
This is because the editor theme is different than the default game one.
it's inconvenient and confusing.
I have to run the game each time.

So I think an option to toggle between editor and game themes would be useful

**Comment by Calinou** [comment]
Are you referring to Theme propagation from parent to children? I see no way to simulate this in the editor. As a workaround, specify your game's Theme resource on the root Control node of every GUI scene.

**Reply by me2beats** [reply]
I mean there is editor theme and there is default game theme and they are different — different colors, styles etc.
And this looks inconsistent when creating a custom theme.
